push-build.sh

```diff
@@ -98,7 +98,7 @@ RELEASE_BUCKET_MIRROR=$FLAGS_bucket_mirror
 [[ $KUBE_GCS_UPDATE_LATEST == "n" ]] && FLAGS_noupdatelatest=1
 
 KUBECTL_OUTPUT=$(cluster/kubectl.sh version --client 2>&1 || true)
-if [[ "$KUBECTL_OUTPUT" =~ GitVersion:\"(${VER_REGEX[release]}\.${VER_REGEX[build]})\", ]]; then
+if [[ "$KUBECTL_OUTPUT" =~ GitVersion:\"((${VER_REGEX[release]}(\.${VER_REGEX[cibuild]})?)\", ]]; then
```

Comment:
Sorry, WIP.  Will PTAL when the time is right.  

WRT to [cibuild], anago checks for an incoming version to build a release from.  This release should be a full CI version string with a +sha.